### ISSUE TITLE
Possible bug in 0.20 release for Google Maps provider

### ISSUE THREAD

**Issue by loic-lopez** [react-community/react-native-maps#2021]
### Is this a bug report?

Yes : 
After upgrading to 0.20 iOS map is displayed instead of Google Map, i rolled back to the 0.19 release and it's working fine.

### Environment

react-native: 0.53.0
react: 16.2.0

11.2 Simulator/iPhone 6 Plus

bug?: react-native-maps: 0.20

working: react-native-maps: 0.19

### Steps to Reproduce

upgrade to 0.20 release

**Comment by rborn** [comment]
@loic-lopez can you give more details, is anything special you did or just simply updating the module?

**Comment by loic-lopez** [comment]
@rborn 
I did : 
`
npm-check-updates -u
npm install
cd ios/
pod update
react-native run-ios
`

**Comment by wbattel4607** [comment]
I'm getting the same issue. iOS is using MapKit instead of Google Maps, even though Google Maps is specified as our preferred provider and was functioning properly in `0.19.0`.

```
<MapView ref={ref => {this.map = ref;}}
    provider={PROVIDER_GOOGLE}
    zoomEnabled={true}
    showsUserLocation={false}
    onPress={Keyboard.dismiss}
    style={styles.container}
    region={this.state.region}
    onRegionChangeComplete={this.onRegionChangeComplete}
    loadingEnabled={false}
    pitchEnabled={false}
    showsCompass={false}
    showsScale={false}
    showsIndoors={false}
    showsIndoorLevelPicker={false}
    customMapStyle={config.map.style}>

    ...

</MapView>
```


```
"react": "^16.2.0",
"react-native": "^0.53.0",
"react-native-maps": "^0.20.0"
```

**Comment by wbattel4607** [comment]
@rborn any ideas? This seems like a pretty big issue. I'm surprised it didn't get caught in testing before being released.

**Comment by wbattel4607** [comment]
@alvelig I meant no disrespect. I understand this is a community effort. That being said, I think it would make more sense to mark brand new untested/lightly-tested versions as release candidate pre-releases. Myself and others would be happy to help test and diagnose issues. The only reason I hadn't done so for this one is because I had no way of knowing if @rborn or another Collaborator was already investigating. If there is a Slack, Discord, etc... that already exists or could be set up to communicate, it could go a long way. Just food for thought.

As for this issue, I have started to pinpoint the problematic commit. So far I have determined that it is somewhere in between 0d540b9b7d3fc2c97262b4a9655ad97202a3992d and f553317fcaa2e3a4889f38c4bb2e215693f3f20c. I should be able to report the exact commit shortly.

**Comment by alvelig** [comment]
@christopherdro do you mind creating a `react-native-maps` slack and sharing invite link in #1807

**Comment by wbattel4607** [comment]
@alvelig I found the culprit.

Commit 9f3c5f6e8be3f95335de20fda073d3edceb2c850 is where this issue started.

**Comment by alvelig** [comment]
@wbattel4607 THX!!! Try latest master, it contains hotfix. 
https://github.com/react-community/react-native-maps/commit/cd868ea7b33a04c8bdd5e909cf134a133b2cb316

**Comment by wbattel4607** [comment]
@alvelig just tried with cd868ea7b33a04c8bdd5e909cf134a133b2cb316 and I'm happy to report that it works!

**Comment by alvelig** [comment]
That was the border case. 
If you did `MapView.PROVIDER_GOOGLE` or `'google'` it would have worked 🙄

**Comment by rborn** [comment]
@christopherdro can we make a push to npm for this hotfix? 🤗

**Comment by alvelig** [comment]
Fixed in 0.20.1
